data/src/main/scala/ch.epfl.scala.index.data/cleanup/GithubRepoExtractor.scala

```diff
@@ -73,7 +73,8 @@ class GithubRepoExtractor extends DefaultJsonProtocol {
         matches(m, s"$groupId $artifactId")
       }.map(_._2).toList
 
-    (fromPoms ++ fromClaims).map{ 
+    /* use claims first - because project indexing uses only the head github repo */
```

Comment:
@MasseGuillaume I analysed the source code a bit to find out what happens with the list of repositories. The Indexing process will only take the *first* github repo and uses that to show related infos on the page.

The github download will take all github repos and fetch the data which actually makes no sense because only the first is relevant for indexing.

I suggest, that we clean up the process for the `GithubRepoExtractor` to only return 1 repo. Also we need to use the claimed repo first and then the defined POM repo. If we do so, we need to cleanup the downloaded github data and delete unused repos.

Would be nice, if we can have a chat about that.